bi_view_editor/models/models.py

```diff
@@ -0,0 +1,171 @@
+# Copyright 2017-2018 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo import SUPERUSER_ID
+from odoo import _, api, models, modules, tools
+from odoo.exceptions import UserError
+
+from odoo.tools import (existing_tables, topological_sort)
+
+_logger = logging.getLogger(__name__)
+
+
+@api.model
+def _bi_view(_name):
+    return _name[0:6] == 'x_bve.'
```

Comment:
if `_name` has less than 6 characters, it will broke here I think.

Reply:
It will not break. If there are less than 6 characters it will take the full string.

Reply:
ok